src/Standards/PSR12/Docs/Properties/ConstantVisibilityStandard.xml

```diff
@@ -0,0 +1,25 @@
+<documentation title="Constant Visibility">
+    <standard>
+    <![CDATA[
+    Visibility must be declared on all class constants if your project PHP minimum version supports class constant visibilities (PHP 7.1 or later).
```

Comment:
Is it "class constants" ? Or all OO constants ? Constants can also be declared on interfaces, traits (PHP 8.2+) and enums, after all.

Also not sure if the "class" needs to be repeated.
```suggestion
    Visibility must be declared on all class constants if your project PHP minimum version supports constant visibilities (PHP 7.1 or later).
```

Reply:
Well, the documentation [states](https://www.php-fig.org/psr/psr-12/#4-classes-properties-and-methods)

> The term "class" refers to all classes, interfaces, and traits.

That's why I opted to use that term.

I added the 'class' to distinguish them from `define`d constants.

Reply:
> Well, the documentation [states](https://www.php-fig.org/psr/psr-12/#4-classes-properties-and-methods)
> 
> > The term "class" refers to all classes, interfaces, and traits.
> 
> That's why I opted to use that term.
> 
> I added the 'class' to distinguish them from `define`d constants.

I fully understand (and support) why it was added, I just wonder if the right term was added. After all, the PSR12 definition is not available from within the PHPCS docs.

Reply:
I've added this to the documentation description, I think it should help to avoid misunderstandings.